internal/http/services/owncloud/ocs/conversions/role.go

```diff
@@ -120,12 +124,19 @@ func (r *Role) WebDAVPermissions(isDir, isShared, isMountpoint, isPublic bool) s
 	if isDir && r.ocsPermissions.Contain(PermissionCreate) {
 		fmt.Fprintf(&b, "CK")
 	}
+
+	if r.CS3ResourcePermissions().DenyGrant && deniable {
```

Comment:
Why is it needed to check for `deniable` here? One should block requests containing the `DenyGrant` in the webdavhandler, possibly returning a BadRequest. We should NOT magically sanitize requests like we do here.

Reply:
this is the ocs part where we can block all deny requests by the config.